### PR TITLE
Introduce manual dispatch.

### PR DESCRIPTION
**Pull request by alexsegura**
A really simple implementation of manual dispatch. 

**Administrator** can assign couriers to deliveries.

manual_dispatch_admin

**Courier** can see assigned deliveries assigned in personal account, and change the status. 

manual_dispatch_courier

This is just a first step before adding more cool stuff, but we need this at least to display it in the native app. 

ping @couleurmate

### CODE REVIEW

File: src/AppBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -172,11 +178,15 @@ order.status.CANCELED: Annulée
...
 delivery.status.WAITING: En attente
-delivery.status.DISPATCHED: Acceptée
+delivery.status.DISPATCHED: Assignée
 delivery.status.PICKED: Récupérée
 delivery.status.DELIVERED: Livrée
 delivery.status.CANCELED: Annulée
 
+delivery.status.DISPATCHED.verb: Assigner
+delivery.status.PICKED.verb: Récupérer
+delivery.status.DELIVERED.verb: Livrer
```
Comment (Atala) [inline src/AppBundle/Resources/translations/messages.fr.yml]: I think it should be `Marquer comme récupérée` et `Marquer comme livrée`

File: src/AppBundle/Resources/views/_partials/Delivery/actions.html.twig [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{% if is_granted('ROLE_ADMIN') and delivery.status == 'WAITING' %}
+  <button type="button" class="btn btn-xs btn-success"
+    data-toggle="modal"
+    data-target="#delivery-modal"
+    data-delivery="{{ delivery.id }}"
+    data-action="dispatch">
+    {% trans %}delivery.status.DISPATCHED.verb{% endtrans %}
+  </button>
+{% elseif is_granted('ROLE_COURIER') and delivery.status == 'DISPATCHED' %}
```
Comment (Atala) [inline src/AppBundle/Resources/views/_partials/Delivery/actions.html.twig]: I think you should also allow admin to do this.

Use-case: an admin is checking all the deliveries at the end of the day and need to update the statuses

Comment (alexsegura) [reply src/AppBundle/Resources/views/_partials/Delivery/actions.html.twig]: Actually `ROLE_ADMIN` inherits from `ROLE_COURIER` 😉 

https://github.com/coopcycle/coopcycle-web/blob/b05e11e7cc14f748a10e3eecd9efc8d8a94a8faa/app/config/security.yml#L13

**Comment by couleurmate** [comment]
Better UX for a courier account to just see the dispatch on the homepage after login.

**Comment by Atala** [comment]
Admin too could be redirected to dashboard page instead of homepage.

-> added to roadmap

**Pull request merged**